<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewPatient</class>
 <widget class="QDialog" name="ViewPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_Form">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="title">
        <string>Patient's info:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_PatientData">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_FirstName">
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leFirstName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_LastName">
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leLastName"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_FatherName">
            <property name="text">
             <string>Father Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leFatherName"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_Address">
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leAddress"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_BirthDate">
            <property name="text">
             <string>Birth Date</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateEdit" name="deBirthDate">
            <property name="displayFormat">
             <string>d/M/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_ArriveDate">
            <property name="text">
             <string>Arrive Date</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDateEdit" name="deArriveDate">
            <property name="displayFormat">
             <string>d/M/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_LeaveDate">
            <property name="text">
             <string>Leave Date</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDateEdit" name="deLeaveDate">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>d/M/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QPushButton" name="btnSetLeaveDate">
            <property name="text">
             <string>Set Leave Date</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdatePatient">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUnstageChanges">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Unstage</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget_ImageGallery"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnNewPhoto">
         <property name="text">
          <string>New Photo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeletePhoto">
         <property name="text">
          <string>Delete Photo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
